### PR TITLE
added index.html & CSS

### PR DESCRIPTION
**Pull request by rancescolon**
HTML and CSS

### CODE REVIEW

File: evergreen/index.html
Comment (Eric-Butcher) [inline evergreen/index.html:1]: Change Ever Green to Evergreen.

Also I would eliminate the sections about the clubs other projects. For now lets just focus on the evergreen greenhouse. Also for the contact us, I would add the discord invite link: https://discord.com/invite/QMHjt65z7a.

File: evergreen/styles.css
Comment (Eric-Butcher) [inline evergreen/styles.css:1]: I don't know CSS enough to authoritatively approve this on my own. But it looks ok so far to my eyes.

**Review by WeirdAlex03 — COMMENTED** [review]
I'm not great on my CSS either but lgtm. With the "Project Evergreen" changes I'll signoff

File: evergreen/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Project-Evergreen</title>
```
Comment (WeirdAlex03) [inline evergreen/index.html]: Just "`Project Evergreen`" - space, not dash

**Review by WeirdAlex03 — CHANGES_REQUESTED** [review]

**Review by WeirdAlex03 — APPROVED** [review]
:shipit:

**Review by Eric-Butcher — APPROVED** [review]

**Pull request merged**